src/sentry/static/sentry/app/components/narrowLayout.tsx

```diff
@@ -22,8 +27,10 @@ class NarrowLayout extends React.Component {
     document.body.classList.remove('narrow');
   }
 
+  private api = new Client();
+
   handleLogout = () => {
-    logout(this.api).then(() => (window.location = '/auth/login'));
+    logout(this.api).then(() => browserHistory.push('/auth/login'));
```

Comment:
Does it matter that `/auth/login` is [currently] a server rendered view?

Reply:
It very well could. I made the change as typescript was complaining that string can't be assigned to window.location, but it can be assigned to window.location.pathname.

Reply:
You could do `window.location.assign()`, that way we could more easily test it too.